doc/src/manual/multi-threading.md

```diff
@@ -213,3 +213,89 @@ therefore a blocking call like other Julia APIs.
 It is very important that the called function does not call back into Julia, as it will segfault.
 
 `@threadcall` may be removed/changed in future versions of Julia.
+
+## Caveats
+
+At this time, most operations in the Julia runtime and standard libraries
+are thread safe.
```

Comment:
I find it somewhat vague to say Julia APIs "_are_ thread safe."  I think the point is that most of those APIs _themselves_ are data race-free.  But, of course, it doesn't prevent users to write data races.  Maybe it'd be clearer to phrase it like:

> At this time, most operations in the Julia runtime and standard libraries can be used in a thread-safe manner, if the user code is data race-free.

?